src/casanovoutils/prec_cov.py

```
@@ -0,0 +1,670 @@
+    for curr_path in pc_df_paths:
+        curr_path = pathlib.Path(curr_path)
+        curr_name = curr_path.stem
+        curr_df = read_dataframe(curr_path)
+        graph_pc.add_series(curr_df, curr_name)
+
+    if out_path is not None:
+        graph_pc.save(out_path)
+
+    try:
+        graph_pc.show()
+    except:
```

Ruff (0.15.6)

[error] Do not use bare `except`

(E722)